pythonforandroid/build.py

```diff
@@ -587,7 +581,7 @@ def build_recipes(build_order, python_modules, ctx):
     return
 
 
-def run_pymodules_install(ctx, modules):
+def run_pymodules_install(ctx, modules, project_dir):
```

Comment:
with that new signature, the integration test are failing at the moment. Could you update to match?
```
Traceback (most recent call last):
  File "setup_testapp_python3_sqlite_openssl.py", line 23, in <module>
    package_data={'testapp_sqlite_openssl': ['*.py', '*.png']}
  File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/home/user/pythonforandroid/bdistapk.py", line 80, in run
    main()
  File "/home/user/pythonforandroid/toolchain.py", line 1051, in main
    ToolchainCL()
  File "/home/user/pythonforandroid/toolchain.py", line 560, in __init__
    getattr(self, args.subparser_name.replace('-', '_'))(args)
  File "/home/user/pythonforandroid/toolchain.py", line 151, in wrapper_func
    build_dist_from_args(ctx, dist, args)
  File "/home/user/pythonforandroid/toolchain.py", line 192, in build_dist_from_args
    build_recipes(build_order, python_modules, ctx)
  File "/home/user/pythonforandroid/build.py", line 579, in build_recipes
    run_pymodules_install(ctx, python_modules)
TypeError: run_pymodules_install() missing 1 required positional argument: 'project_dir'
travis_time:end:0195f4fa:start=1549223130899523065,finish=1549223892350143515,duration=761450620450
[0K[31;1mThe command "docker run p4a /bin/sh -c "$COMMAND"" exited with 1.
```

Reply:
@AndreMiras oops, a leftover from the setup.py part I forgot to remove

Reply:
resolving since this is fixed now :+1: thanks for pointing out